Logs/README.md

```diff
@@ -0,0 +1,9 @@
+# Logs
+```
+{
+  'type': string,
+  'timestamp': timestamp,
+  'meta': {},
+  'body': {},
```

Comment:
'type', 'timestamp' has `type` as values where 'meta', 'body' has `{}`

It would be consist if we use
'meta': object,
'body': object

Reply:
Ok will make the change. I copied that from user model.